oracle/config/collected-signatures-watcher.config.js

```diff
@@ -2,11 +2,20 @@ const baseConfig = require('./base.config')
 
 const id = `${baseConfig.id}-collected-signatures`
 
-module.exports = {
+const config = {
   ...baseConfig.bridgeConfig,
   ...baseConfig.homeConfig,
   event: 'CollectedSignatures',
   queue: 'foreign-prioritized',
   name: `watcher-${id}`,
   id
 }
+
+if (baseConfig.id === 'erc-native') {
+  config.accessLists = {
+    blockList: '/mono/oracle/access-lists/block_list.txt',
```

Comment:
Does it make sense to specify the path in the configuration parameters provided in `.env`? E.g. two parameters could be introduced `ORACLE_HOME_TO_FOREIGN_ALLOWANCE_LIST` and `ORACLE_HOME_TO_FOREIGN_BLOCK_LIST`. If the parameter is not configured, don't try to read the file at all.

Reply:
I think it does not make too much sense. In any case, we are running our services in docker containers, so these file names are only for the internal usage within a container.

Reply:
But if you run it locally by yarn and without the docker, you need to create directory `/mono/oracle/access-lists` on your system.

Reply:
Ok, introduced two environment variables for that.